Application/EdFi.Ods.Features/OpenApiMetadata/Strategies/ResourceStrategies/OpenApiMetadataUiDescriptorOnlyStrategy.cs

```diff
@@ -10,15 +10,15 @@
 
 namespace EdFi.Ods.Features.OpenApiMetadata.Strategies.ResourceStrategies
 {
-    public class SwaggerUiDescriptorOnlyStrategy : IOpenApiMetadataResourceStrategy
+    public class OpenApiMetadataUiDescriptorOnlyStrategy : IOpenApiMetadataResourceStrategy
```

Comment:
lets reformat this whole document please

Reply:
this comment was not completed

Reply:
Fixed